### DISCUSSION TITLE
Automatic photos display

### DISCUSSION THREAD

**Discussion by lihoooooo** [gopro/labs#913] [Feature Requests]
Hi,

I'm looking to execute an automatic photos display after a hot air ballooning flight to show to passengers.
I cannot find this function. I have around 70 photos to show but I would like it display each photo every 2 seconds in application Quick on iPad. Any ideas if we could do that ?

Thanks ! Regards,

Laurent

**Comment by dnewman-gpsw** [comment]
This is not something Labs firmware can help with. Sounds doable though, but outside this forum's expertise.
